common/models/access-token.js

```diff
@@ -7,6 +7,9 @@
  * Module Dependencies.
  */
 
+var SG = require('strong-globalize');
+var g = SG();
```

Comment:
Ideally, we would like to do that.

However, strong-globalize does 2 things:
1. Offline extraction slt-globalize -e
2. runtime replacement such as g.f

Moving that code piece to one file breaks offline extraction where it scans the source-code to see which lines need translation.
The module that requires it will have the strings extracted. If we move them to one file, then we won't be able to extract the strings needed.